Updating to include when

carolyncole opened the pull request:
This document include the how already.  Just adding a statement on when to the top.

hackartisan reviewed (approved):
Thanks for these extra link!

carolyncole committed:
Updating to include when

This document include the how already.  Just adding a statement on when to the top.
